src/form.svelte

```diff
@@ -12,13 +12,36 @@
   let saveState = "";
   let errorMessage = "";
   let availableTagNames = []
+  let formTitle = "";
 
   async function init() {
     const tabInfo = await getCurrentTabInfo();
     url = tabInfo.url;
     titlePlaceholder = tabInfo.title;
     const availableTags = await getTags().catch(() => [])
     availableTagNames = availableTags.map(tag => tag.name)
+
+   initCurrentBookmarkData();
+  }
+
+  async function initCurrentBookmarkData() {
+    formTitle = 'Searching for existing Bookmarks';
+
+     var bookmarkSearch = await lookForCurrentBookmark(url)
+        .catch(() => []);
+
+    if ( bookmarkSearch && bookmarkSearch.length > 0 ) {
+        var temp = bookmarkSearch[0];
+        formTitle = 'Edit Bookmark';
```

Comment:
I feel this is not enough to make people aware that they are effectively editing an existing bookmark. Users are used to looking at the tags input first, as that has auto-focus, and the URL above usually doesn't need to be changed. After that they only proceed reading downwards. It's unlikely that they will notice a difference of 1 character in the title. Some might wonder why there are already tags, but they could also dismiss that as a bug, and just enter a new value.

I think this comment from the previous PR would still be good to have: https://github.com/sissbruecker/linkding-extension/pull/10#discussion_r766157594